ci-operator/step-registry/openshift-extended/web-tests/ext-oidc-cli-login/openshift-extended-web-tests-ext-oidc-cli-login-commands.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
```

Comment:
@fxierh  I am ok for this step. thanks

reminder to change console-test-azure-external-oidc.sh  to only run that one case, not per ts file.

Reply:
Got it, thanks for the reminder